server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/ElasticSearchMailboxModule.java

```diff
@@ -129,6 +132,8 @@ private Client connectToCluster(ElasticSearchConfiguration configuration) {
             configuration.getWriteAliasMailboxName(),
             configuration.getIndexMailboxName());
 
+        QuotaSearchIndexCreationUtil.prepareDefaultClient(client);
```

Comment:
what is the purpose of that ?

Reply:
The purpose is to create indexes & aliases for Quota Ratio search.

This will be more explicit once we will read it from configuration.